cumulusci/core/config/universal_config.py

```diff
@@ -82,3 +82,7 @@ def _load_config(self):
                 "global_config": UniversalConfig.config_global,
             }
         )
+
+    @property
+    def cache_dir(self):
+        return self.config_global_local_path
```

Comment:
@prescod config_global_local_path was renamed to config_global_path as part of the universal/global/local clarification. And be aware that this is one of the ones that MetaCI sets to None: https://github.com/SFDO-Tooling/MetaCI/blob/master/metaci/cumulusci/config.py#L52

Reply:
This is one of those things that makes my head hurt. What is the circumstance where this code gets invoked and how can I emulate or trigger it?

We look at the universal config if we don't have a project config.

But the universal config is a subclass of BaseTaskFlowConfig, so its presumably intended to be used when invoking Tasks or Flows. But as of August 2020, Tasks and Flows always run within the context of Projects...

Does this method exist just to future-proof in case we one-day allow tasks and flows to run outside of projects?

Or is there such a context right now? MetaDeploy or Metecho or ...

Reply:
I'm leaning towards just deleting this method unless you know of a case where the UniversalConfig is the project_config in a context that needs access to the describe schema or other info we might cache about orgs.

Reply:
And then open_cache would move into project_config and cache_dirs would be a project_config-only feature.

Reply:
@prescod I'm fine with making it a project_config-only feature. I'm pretty sure that there isn't currently a place where tasks and flows can be run without having a project_config as context (actually I'm quite certain, since project_config is the first thing passed when instantiating a task).

In theory we could make it so that tasks and flows can be run outside the context of a project if they are defined in universal or global config and interact with no org or a global org. But it's probably cleanest to keep this out of UniversalConfig until we're actually considering that possibility closely.

Reply:
b3adc74